src/SprykerEco/Zed/AmazonQuicksight/Presentation/_partials/quicksight-analytics.twig

```diff
@@ -26,12 +26,36 @@
             <script src="{{ assetsPath('js/spryker-zed-quicksight-analytics-gui-main.js') }}"></script>
         {% endif %}
     {% else %}
-        {{ 'No Analytics permission has been granted to the current user.' | trans }}
+        <div class="alert alert-info">
+            {{ 'No Analytics permission has been granted to the current user.' | trans }}
+        </div>
     {% endif %}
 {% elseif isAssetBundleInitializationInProgress %}
     <div class="alert alert-info" role="alert">{{ 'Spryker Analytics is being activated. Please wait 1-2 minutes and reload the page.' | trans }}</div>
 {% else %}
-    <h3>{{ 'Spryker Analytics is not enabled yet' | trans }}</h3>
-    {{ 'You can enable Spryker Analytics to view and modify default dashboards with the ability to share them with other users. ' | trans }}
-    {{ viewActionButton(url('/amazon-quicksight/analytics/enable'), 'Enable Analytics' | trans) }}
+    <div class="analytics">
+        <div class="analytics__icon"><i class="fa fa-chart-bar"></i></div>
+
+        <div class="analytics__content">
+            <h3 class="analytics__heading">{{ 'Spryker Analytics is not enabled yet' | trans }}</h3>
+            <p>{{ 'You can enable Spryker Analytics to view and modify default dashboards with the ability to share them with other users. ' | trans }}</p>
+        </div>
+
+        <div class="analytics__button">
+            {{ viewActionButton(url('/amazon-quicksight/analytics/enable'), 'Enable Analytics' | trans) }}
+        </div>
+    </div>
 {% endif %}
+
+<div class="analytics">
```

Comment:
Suggestion create macros and reuse it, cause you have to identical html blocks

Reply:
Oh, it was just unused code. Removed it, thanks.